python/CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ set_target_properties(pyf3d PROPERTIES
   OUTPUT_NAME "f3d"
   )
 
+if (DEFINED SKBUILD)
+  # If we are building a python wheel, we import _f3d.* in __init__.py
+  target_compile_definitions(pyf3d PRIVATE F3D_PYTHONLIB_NAME=_f3d)
```

Comment:
Is it standard to add a `_` as a prefix? Is it making sense to add it even if we build a non-wheel wrapper?

Reply:
Would it be better to use a configure_file somewher ?